src/js/components/TaskListItemComponent.jsx

```diff
@@ -228,6 +204,49 @@ var TaskListItemComponent = React.createClass({
     </a>);
   },
 
+  getPortQuickLink: function (index, port, ip) {
+    const SSHFORWARD_LABEL_NAME = "sshforward";
+
+    var portDefinitions = this.props.portDefinitions;
+    var props = this.props;
+    var ports = props.task.ports;
+    const scheme = ServiceSchemeUtil
+      .getServiceSchemeFromLabels(props.labels, index);
+    const name = getPortDecoration(portDefinitions, index, port);
+    if (name === SSHFORWARD_LABEL_NAME) {
+      var portFwConfig = ports.filter(
+        (port, idx) =>
+          getPortDecoration(portDefinitions, idx, port)
+          !== SSHFORWARD_LABEL_NAME)
+        .map(function (portNumber) {
+          return "-L " + portNumber + ":127.0.0.1:" + portNumber;
+        }).join(" ");
+      var app = AppsStore.getCurrentApp(props.appId);
+      var command = `ssh -N ${portFwConfig} -p ${port} ${app.user}@${ip}`;
+      return (
+        <TooltipComponent className="top" message="Click to copy ssh
+             forward command">
+          <a key={`${ip}:${port}`}
+             className="text-muted"
+             onClick={() => {
+               navigator.clipboard.writeText(command);
+             }}
+             target="_blank">
+            {name}
+          </a>
+        </TooltipComponent>
+      );
+    } else {
+      return (
+        <a className="text-muted"
```

Comment:
Not sure if this has an importance but you did not set the key here

Reply:
i'll check but I think we can remove them as we're not in a list context

Reply:
There was a key before when there was a list of port, this is why I'm asking here. I'm approving the PR so that you can merge directly after checking :)

Reply:
Removing seems all good, thanks for the review !